enzyme/CMakeLists.txt

```diff
@@ -70,17 +71,29 @@ message("CMAKE_PREFIX_PATH " ${CMAKE_PREFIX_PATH})
 
 find_package(LLVM REQUIRED CONFIG)
 
+if (NOT LLVM_ENABLE_PLUGINS)
+  message("LLVM_ENABLE_PLUGINS OFF")
+  message("Setting ENZYME_ENABLE_PLUGINS OFF")
+  set(ENZYME_ENABLE_PLUGINS OFF)
+endif()
+
+if ((NOT ENZYME_EXTERNAL_SHARED_LIB) AND (NOT ENZYME_ENABLE_PLUGINS))
+  message("ENZYME_EXTERNAL_SHARED_LIB OFF")
+  message("ENZYME_ENABLE_PLUGINS OFF")
+  message(SEND_ERROR "You have selected a built without creating the Enzyme plugin, nor Enzyme as a shared library -- and nothing will be built.")
```

Comment:
Selected a build not a built

Reply:
not on a computer anymore, can you update it?